tests/Integration/Model/TagTest.php

```diff
@@ -748,6 +748,116 @@ public function testDeleteContainerTag()
         $this->assertSame(1, $count);
     }
 
+    public function testPauseContainerTag()
+    {
+        $params = ['customHtml' => '<div>foo</div>'];
+        $this->addContainerTag($this->idSite, $this->containerVersion1, null, 'v1', $params);
+        $idTag3 = $this->addContainerTag($this->idSite, $this->containerVersion1, null, 'v2', $params);
+        $this->addContainerTag($this->idSite2, $this->containerVersion1, null, 'v2', $params, [$this->idTrigger4]);
+        $this->addContainerTag($this->idSite2, $this->containerVersion1, null, 'v3', $params, [$this->idTrigger4]);
+        $this->addContainerTag($this->idSite, $this->containerVersion2, null, 'v4', $params, [$this->idTrigger2]);
+
+        $this->model->setCurrentDateTime('2019-03-04 03:03:03');
+
+        $this->assertCount(3, $this->model->getContainerTags($this->idSite, $this->containerVersion1));
```

Comment:
Quick question: I see 2 addContainerTag calls for the idSite + containerVersion1 combination but it's doing assertCount 3. There's another from outside this function or something?

Reply:
@araichyk Its due to this [initial_tag](https://github.com/matomo-org/tag-manager/blob/5.x-dev/tests/Integration/Model/TagTest.php#L75) being added on `setUp`